packages/discount-code-importer/src/main.js

```diff
@@ -1,3 +1,221 @@
-export default function findSum (a, b) {
-  return a + b
+/* @flow */
+import type {
+  LoggerOptions,
+  ChunkOptions,
+  CodeDataArray,
+  CodeData,
+  ConstructorOptions,
+  Summary,
+} from 'types/discountCodes'
+import type {
+  Client,
+  AuthMiddlewareOptions,
+  SyncAction,
+ } from 'types/sdk'
+import npmlog from 'npmlog'
+import Promise from 'bluebird'
+import _ from 'lodash'
+import { createClient } from '@commercetools/sdk-client'
+import { createRequestBuilder } from '@commercetools/api-request-builder'
+import { createAuthMiddlewareForClientCredentialsFlow }
+  from '@commercetools/sdk-middleware-auth'
+import { createHttpMiddleware } from '@commercetools/sdk-middleware-http'
+import { createUserAgentMiddleware }
+  from '@commercetools/sdk-middleware-user-agent'
+import { createSyncDiscountCodes } from '@commercetools/sync-actions'
+import { version } from '../package.json'
+
+export default class DiscountCodeImport {
+  // Set flowtype annotations
```

Comment:
Yes it is. you can read more about it [here](https://flow.org/en/docs/types/classes/#toc-class-fields-properties)